src/plugins/intel_cpu/src/cpu_map_scheduling.cpp

```diff
@@ -79,19 +79,29 @@ bool get_cpu_pinning(bool& input_value,
     int num_sockets = get_default_latency_streams(latency_threading_mode);
     bool latency = num_streams <= num_sockets && num_streams > 0;
 
-    if (proc_type_table[0][EFFICIENT_CORE_PROC] > 0 &&
-        proc_type_table[0][EFFICIENT_CORE_PROC] < proc_type_table[0][ALL_PROC]) {
-        result_value =
-            input_changed
-                ? input_value
-                : ((latency || bind_type == threading::IStreamsExecutor::ThreadBindingType::NUMA) ? false : true);
+    if (input_changed) {
```

Comment:
this function becomes too complicated and the main problem it combines cpu_pinning initialization logic based on plugin needs with limitations caused by affinity API (like proc_type_table.size() > 1).
I would propose to split the logic on two function: first will combine only plugin logic while second will provide is_pinning_supported API from thread_affinity.hpp
Anyway the comment is minor and can be resolved separately.

Reply:
Ok, I will solve it later. Thank you.